apps/ui/src/components/ProposalVoteBasic.vue

```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
+import { Choice } from '@/types';
+
+withDefaults(
+  defineProps<{
+    sendingType: Choice | null;
+    size: number;
+  }>(),
+  { size: 48 }
+);
+
+const emit = defineEmits<{
+  (e: 'handleVoteClick', value: Choice);
```

Comment:
```suggestion
  (e: 'voteClick', value: Choice);
```

We tend to call handler function with `handle` prefix, but emit names don't have that prefix.